### PR TITLE
fix: handle input-object-type definitions

### PR DESCRIPTION
**Pull request by YevheniiMelikov**
This pr will add handler for `InputObjectTypeDefinition` in visitor config.

### CODE REVIEW

File: tests/typescript-mock-data.spec.ts [hunk 1]
```diff
@@ -18,6 +18,11 @@ const testSchema = buildSchema(/* GraphQL */ `
     type Query {
         user: User!
     }
+
+    input QueryInput {
```
Comment (ardeois) [inline tests/typescript-mock-data.spec.ts]: are you sure the name `QueryInput` makes sense? it's an input type for mutations not queries.
Maybe name it `InputType`, or like what was done with `Avatar` and `User`, try to find a name for a real life example

Comment (YevheniiMelikov) [reply tests/typescript-mock-data.spec.ts]: Yeah, you are right
Updated

File: tests/__snapshots__/typescript-mock-data.spec.ts.snap [hunk 2]
```diff
@@ -33,6 +49,22 @@ export const anAvatar = (overrides?: Partial<Avatar>): Avatar => {
     };
 };
 
+export const aMutation = (overrides?: Partial<Mutation>): Mutation => {
+    return {
+        updateUser: aUser(),
+        ...overrides
+    };
+};
```
Comment (ardeois) [inline tests/__snapshots__/typescript-mock-data.spec.ts.snap]: I don't think we should generate mock data for mutations.
We already exclude `Query` type
See here

Comment (YevheniiMelikov) [reply tests/__snapshots__/typescript-mock-data.spec.ts.snap]: Yeah, makes sense
Fixed

**Comment by ardeois** [comment]
@YevheniiMelikov can you create another commit, I've activated Circle CI for fork pull requests.
If this is not triggering a build, could you try to open a new PR please?
Sorry for the inconvenience but I don't want to merge a PR if the tests are not run by CI

**Comment by YevheniiMelikov** [comment]
@ardeois Done, merge it, please

**Comment by ardeois** [comment]
Thanks !

**Comment by ardeois** [comment]
@YevheniiMelikov Version 0.1.2 published !
https://www.npmjs.com/package/graphql-codegen-typescript-mock-data/v/0.1.2

**Pull request merged**